examples/network/imageSelected/imageSelected.html

```diff
@@ -0,0 +1,70 @@
+<html>
+<head>
+  <title>Network | Selected/Unselected Image</title>
+
+  <script type="text/javascript" src="../../../dist/vis.js"></script>
+  <link href="../../../dist/vis-network.min.css" rel="stylesheet" type="text/css" />
+
+  <style type="text/css">
+    body {
+      font: 10pt arial;
+    }
+    #mynetwork {
+      width: 600px;
+      height: 600px;
+      border: 1px solid lightgray;
+      background-color:#333333;
+    }
+  </style>
+  <script src="../../googleAnalytics.js"></script>
+</head>
+
+<body>
+<div id="mynetwork"></div>
+<script type="text/javascript">
+  // create an array with nodes
+  var nodes = new vis.DataSet([
+    {id: 1, label: 'Node 1'},
+    {id: 2, label: 'Node 2'},
+    {
+      id: 11,
+      font:{color: '#CCC', size:30},
+      size:40,
+      label: 'Select me!',
+      // shape: 'circularImage',
+      shape: 'image',
+      brokenImage: './broken-image.png',
+      // image: './unselected.svg',
+      image: {
+        unselected: './unselected.svg',
+        selected: './selected.svg',
+      },
+      shapeProperties: {
+        borderDashes:[15,5],
+        interpolation: false,
+      }
+    }
+  ]);
+
+  // create an array with edges
+  var edges = new vis.DataSet([
+    {from: 1, to: 11},
+    {from: 1, to: 2},
+  ]);
+
+  // create a network
+  var container = document.getElementById('mynetwork');
+
+  // provide the data in the vis format
+  var data = {
+    nodes: nodes,
+    edges: edges
+  };
+  var options = {};
```

Comment:
The broken-image should be applied to all nodes. (I also added a randomSeed so always have a nice alignment).
```js
  var options = {
    layout:{
      randomSeed: 5
    },
    nodes: {
      brokenImage: './broken-image.png',
    }
  };
``